app/models/identity.rb

```diff
@@ -73,7 +73,10 @@ class Identity < ApplicationRecord
   has_many :protocol_service_requests, through: :protocols, source: :service_requests
   has_many :studies, -> { where("protocols.type = 'Study'")}, through: :project_roles, source: :protocol
 
-  cattr_accessor :current_user
```

Comment:
Any reason this was removed?

Reply:
@amcates that was not intentional, I will add it back then re-push to my pull request.